### ISSUE TITLE
Add SCF Helper Release to release images

### ISSUE THREAD

**Issue by gaktive** [SUSE/kubecf#35]
Build SCF Helper Release with the post-deployment job initially.

Investigate if we need all bits of post-deployment.

**Comment by fargozhu** [comment]
@f0rmiga do we need this within kubecf context?

**Comment by f0rmiga** [comment]
@fargozhu No. We discussed this on the last sprint planning. I need to do a clean-up before closing this issue.
